spec/controller_services/shopping_list_items_controller/create_service_spec.rb

```diff
@@ -13,14 +13,17 @@
     subject(:perform) { described_class.new(user, shopping_list.id, params).perform }
 
     let(:user) { create(:user) }
+    let(:game) { create(:game, user: user) }
 
     context 'when all goes well' do
-      let!(:aggregate_list) { create(:aggregate_shopping_list, user: user) }
-      let!(:shopping_list) { create(:shopping_list, user: user, aggregate_list: aggregate_list) }
+      let!(:aggregate_list) { create(:aggregate_shopping_list, game: game) }
+      let!(:shopping_list) { create(:shopping_list, game: game, aggregate_list: aggregate_list) }
       let(:params) { { description: 'Necklace', quantity: 2, notes: 'Hello world' } }
 
       before do
-        allow(user.shopping_lists).to receive(:find).and_return(shopping_list)
+        user_lists = user.shopping_lists
```

Comment:
Like in another recent PR, this broke for some reason and adding this code fixed it.